src/videojs-http-streaming.js

```diff
@@ -1062,11 +1062,34 @@ class VhsHandler extends Component {
 
     this.player_.tech_.on('keystatuschange', (e) => {
       if (e.status === 'output-restricted') {
```

Comment:
If we change this to:

```
      if (e.status !== 'output-restricted') {
        return;
      }
```

we can remove a level of indentation.

Reply:
Yup, should have done that.